src/main/java/org/spongepowered/api/CatalogTypes.java

```diff
@@ -253,7 +252,6 @@
     public static final Class<HorseStyle> HORSE_STYLE = HorseStyle.class;
 
     @Deprecated public static final Class<HorseVariant> HORSE_VARIANT = HorseVariant.class;
-
```

Comment:
Huh weird, I think `sortClassFields` might have done this for some reason. Maybe because of the annotation?